config.go

```diff
@@ -107,16 +107,27 @@ func (sc *SafeConfig) ReloadConfig(configFile string) error {
 		log.Infoln("Found ipmi user env")
```

Comment:
sc.C.Credentials["default"] should not be default but the job name. (like netbox case)